src/types/shopItem.ts

```diff
@@ -0,0 +1,47 @@
+type Price = {
+  selling_price: number;
+  original_price: number;
+  discounted: boolean;
+  discount_percentage: number;
+  discount_amount: number;
+};
+
+export type ShopItem = {
+  id: string;
+  name: string;
+  thumbnail: string;
+  price: Price;
+  tags: [
+    {
+      id: string;
+      name: string;
+      color: string;
+      tag_group: string;
+    },
+  ];
```

Comment:
[must]
型定義の書き方は以下の方が良いと思いました！
いくつかありますので、全て修正していただければと思います。

```suggestion
  tags: {
      id: string;
      name: string;
      color: string;
      tag_group: string;
  }[],
  ;
```

Reply:
修正いたしました！ありがとうございます。
[5c09c40](https://github.com/tosssssy/yumeshop-frontend/pull/4/commits/5c09c405109a7dd4f28045b2c8b79fe787f79a23)